src/ch/imedias/rsccfx/view/RsccSupportPresenter.java

```diff
@@ -83,22 +78,21 @@ public String validationImage(String token) {
    */
   private void attachEvents() {
 
-    view.tokenTxt.setOnKeyPressed(event -> {
+    view.tokenTxt.setOnKeyReleased(event -> {
       view.isValidImg.setImage(new Image(validationImage(view.tokenTxt.getText())));
```

Comment:
Maybe it would make sense here to make the validationImage method return an image instead of a String? This way we can simply include the method call here without creating a new image